### ISSUE TITLE
[BUG] Reopen JS template literal close indent issue #586

### ISSUE THREAD

**Issue by jpradelle** [yeonjuan/html-eslint#614]
Hi sorry to reopen this one #586 again after confirming it was fixed.

I found a new case in issue.

You fixed  as asked in issue #586 for this one, this is OK
```javascript
html`
  <div>
    ${when(
        true,
        () => html`
          <p>ok</p>
        `)} // ok
  </div>
`
```

But for this one, I would expect that:
```javascript
html`
  <div>
    ${when(
        true,
        () => {
          // Some code
          return html`
            <p>not ok</p>
          `;
        })} // expected to be OK and requested by @stylistic/indent
  </div>
`
```

And here is is asked by html-eslint:
```javascript
html`
  <div>
    ${when(
        true,
        () => {
          // Some code
          return html`
            <p>not ok</p>
          `;
    })} // Requested by html-eslint
  </div>
`
```
I think the issue is that the line starts with `}` but without checking that this one is not the closing of `${`

Playground link (Warning, playground is in version `v0.59.0` and you fixed first case in `v0.60.0`) https://html-eslint.org/playground#eyJjb2RlIjoiaHRtbGBcbiAgPGRpdj5cbiAgICAke3doZW4oXG4gICAgICAgIHRydWUsXG4gICAgICAgICgpID0+IGh0bWxgXG4gICAgICAgICAgPHA+b2s8L3A+XG4gICAgICAgIGApfVxuICAgICR7d2hlbihcbiAgICAgICAgdHJ1ZSxcbiAgICAgICAgKCkgPT4ge1xuICAgICAgICAgIC8vIFNvbWUgY29kZVxuICAgICAgICAgIHJldHVybiBodG1sYFxuICAgICAgICAgICAgPHA+bm90IG9rPC9wPlxuICAgICAgICAgIGA7XG4gICAgICAgIH0pfVxuICA8L2Rpdj5cbmAiLCJjb25maWciOnsicnVsZXMiOnsiQGh0bWwtZXNsaW50L2luZGVudCI6WyJlcnJvciIsMl19fSwibGFuZ3VhZ2UiOiJqYXZhc2NyaXB0In0=

**Comment by yeonjuan** [comment]
Hi @jpradelle  I've released an updated version as an alpha ([0.62.0-alpha.0](https://www.npmjs.com/package/@html-eslint/eslint-plugin/v/0.62.0-alpha.0)). Would you mind taking a look?

**Comment by jpradelle** [comment]
Hi thanks a lot, this looks good, I ran it on 3 large projects without unexpected issues.

**Comment by yeonjuan** [comment]
@jpradelle Thank you for checking!
